### ISSUE TITLE
Some TikTok videos report video not found in feed and not able to download the watermark-free version

### ISSUE THREAD

**Issue by hanimourra** [yt-dlp/yt-dlp#6483]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a site-specific feature
- [X] I've verified that I'm running yt-dlp version **2023.03.04** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Canada

### Example URLs

yt-dlp -vU --no-mtime --no-continue --proxy https://154.12.116.74:8800 --no-cache-dir -F https://www.tiktok.com/@moxypatch/video/7206382937372134662 --extractor-args "tiktok:manifest_app_version=25.9.0;app_version=2590160"

### Provide a description that is worded well enough to be understood

Some TikTok videos report video not found in feed and not able to download the watermark-free version

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU --no-mtime --no-continue --proxy https://154.12.116.74:8800 --no-cache-dir -F https://www.tiktok.com/@moxypatch/video/7206382937372134662 --extractor-args "tiktok:manifest_app_version=25.9.0;app_version=2590160"

[debug] Command-line config: ['-vU', '--no-mtime', '--no-continue', '--proxy', 'https://154.12.116.74:8800', '--no-cache-dir', '-F', 'https://www.tiktok.com/@moxypatch/video/7206382937372134662', '--extractor-args', 'tiktok:manifest_app_version=25.9.0;app_version=2590160']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.03.04 [392389b7d] (zip)
[debug] Python 3.8.9 (CPython x86_64 64bit) - Linux-4.15.0-173-generic-x86_64-with-glibc2.17 (OpenSSL 1.0.2n  7 Dec 2017, glibc 2.27)
[debug] exe versions: ffmpeg 5.0.1-static (setts), ffprobe 5.0.1-static
[debug] Optional libraries: no_Cryptodome-None, sqlite3-2.6.0
[debug] Proxy map: {'http': 'https://154.12.116.74:8800', 'https': 'https://154.12.116.74:8800'}
[debug] Loaded 1786 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.03.04, Current version: stable@2023.03.04
Current Build Hash: 91cad9f121c1f6f0a81b747415c46ecba0ff331ed38cc6433040b4ac7b6e15ca
yt-dlp is up to date (stable@2023.03.04)
[TikTok] Extracting URL: https://www.tiktok.com/@moxypatch/video/7206382937372134662
[debug] [TikTok] Imported app version combo from extractor arguments
[TikTok] 7206382937372134662: Downloading video feed
WARNING: [TikTok] 7206382937372134662: Unable to find video in feed; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U; trying with webpage
[TikTok] 7206382937372134662: Downloading webpage
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 7206382937372134662:
ID EXT RESOLUTION │ PROTO │ VCODEC  ACODEC
───────────────────────────────────────────
0  mp4 576x1024   │ https │ unknown unknown
```

**Comment by bashonly** [comment]
This issue is closely related to #4992, but not quite a duplicate. #4992 is tracking videos where the audio has been taken down by TikTok, and as a result the video is not available via the feed API endpoint *or* the webpage, and cannot be downloaded by yt-dlp at all.

This issue tracks videos that are unavailable via the feed endpoint due to an unknown reason, yet are still available via the webpage.

**Comment by imawin954** [comment]
The issue on downloading user profiles #3776 is locked to collaborators only so I can't post there. But there seems to be about a 50/50 chance that when trying to download a user's profile with the link to their account, it will be able to grab all their videos. This includes videos with their sound removed, and videos that only download with watermarks when using a link to t he video will download without watermark. However there is another issue, and it's been around since the last time downloading profiles worked, it will occasionally download the lowest quality version. This also happens occasionally when downloading a single video, but a lot more frequent when downloading a profile.

**Comment by bashonly** [comment]
apparently fixed by cb61e20c266facabb7a30f9ce53bd79dfc158475, but can re-open this if anyone else has another example
